test/config.ts

```diff
@@ -7,10 +7,17 @@ if (process.env.LOAD_ENV) {
     path: path.join(__dirname, '../.env'),
   });
 }
-const DEFAULT_CLIENT_ID = 'dummy-acme-client-id';
-const DEFAULT_CLIENT_SECRET = 'dummy-acme-client-secret';
+const DEFAULT_CYBEREASON_ID = 'dummy-acme-client-id';
+const DEFAULT_CYBEREASON_SECRET = 'dummy-acme-client-secret';
+const DEFAULT_CYBEREASON_HOST = 'dummy-acme-client-host';
+const DEFAULT_CYBEREASON_PORT = 'dummy-acme-client-port';
 
 export const integrationConfig: IntegrationConfig = {
-  clientId: process.env.CLIENT_ID || DEFAULT_CLIENT_ID,
-  clientSecret: process.env.CLIENT_SECRET || DEFAULT_CLIENT_SECRET,
+  cybereasonId: process.env.CYBEREASON_ID || DEFAULT_CYBEREASON_ID,
+  cybereasonPassword:
+    process.env.CYBEREASON_SECRET || DEFAULT_CYBEREASON_SECRET,
+  cybereasonHost:
+    process.env.DEFAULT_CYBEREASON_HOST || DEFAULT_CYBEREASON_HOST,
+  cybereasonPort:
+    process.env.DEFAULT_CYBEREASON_PORT || DEFAULT_CYBEREASON_PORT,
```

Comment:
Host and port is user dependent.

Reply:
Yes, the end-user/customer will need to provide to the integration their host and port :100:.